disttask/loaddata/scheduler.go

```diff
@@ -113,40 +121,44 @@ func (s *ImportScheduler) OnSubtaskFinished(ctx context.Context, subtaskMetaByte
 	logutil.BgLogger().Info("OnSubtaskFinished", zap.Any("taskMeta", s.taskMeta))
 	var subtaskMeta SubtaskMeta
 	if err := json.Unmarshal(subtaskMetaBytes, &subtaskMeta); err != nil {
```

Comment:
maybe we can use generic type at implementation 🤔 if the serializing and deserializing is needed, framework can use json.Unmarshal to get a SubtaskMeta